### ISSUE TITLE
release?

### ISSUE THREAD

**Issue by Njasa2k** [krennic999/STAR#1]

**Comment by krennic999** [comment]
Will, around a month, we need to follow the company's open-source process.

**Comment by ucasyjz** [comment]
Great, Can't wait to try it out!

**Comment by daiyixiang666** [comment]
Will you guys also release the weights? Thanks!

**Comment by krennic999** [comment]
Yes, please wait for a while, thanks.

**Comment by ucasyjz** [comment]
Will the training code be released

**Comment by daiyixiang666** [comment]
Do you have some evaluation result about the 256x256 VAR like the FID in animal or other case in the MJD-30k

**Comment by daiyixiang666** [comment]
Beside，do you think the result that you generate are darker than usual image ?

**Comment by krennic999** [comment]
We evaluated on ImageNet-val under 256 cases, and the recon PSNR is around 22, which seems to be better than VQGAN.
The brightness of the generated dataset is caused by the dataset used for training. Fine-tuning on a small subset will alleviate this issue.

**Comment by daiyixiang666** [comment]
yes,thanks , but I means the VAR not the VAE result, do you have the FID score of the VAR in Per-category FID on MJHQ-30K? And Really thanks for your reply again!

**Comment by krennic999** [comment]
Well, the per-category FID on MJHQ can be found in Fig.2 in our mainpaper, for specific value, refer to table below:

![8061720005699_ pic](https://github.com/krennic999/STAR/assets/157558015/41e139f9-ad96-4dac-9e5f-a0f214683c74)

**Comment by daiyixiang666** [comment]
Thanks a lot, I would say that it is really impressive to see that it has such low FID score!!!

**Comment by daiyixiang666** [comment]
Do you guys add the qk normalization

**Comment by krennic999** [comment]
We do not have qk normalization, but probably will in the next version. The visual auto-regression paradigm proposed by VAR is full of potential, and we are currently working on exploring it for more stable and amazing results.

**Comment by daiyixiang666** [comment]
So looking forward to the code and ckpt hhhhh

**Comment by nunbuzor** [comment]
Hi @krennic999, I was wondering if there were any updates on the release timeline? Was it still scheduled for next week, or have there been any changes or delays? Looking forward to hearing back from you!

**Comment by krennic999** [comment]
Hi, we apologize for the inconvenience. After our discussions, we have determined that the current version, due to issues with VQVAE and other factors, is not stable enough for practical applications. We may release a revised version, including modified VQVAE and 1024 generation later this year.

**Comment by krennic999** [comment]
However, we will do our best to answer any questions about this project. Thank you for your interest.

**Comment by daiyixiang666** [comment]
Is the unstable means although the model can achieve a lower FID score, but the generative image is not as stable as in diffusion model?

**Comment by daiyixiang666** [comment]
I also do my own VAR t2i training.  I find that it is somehow like 抽奖

**Comment by Ccioud** [comment]
Hi,@krennic999，can I ask when is the approximate release date?

**Comment by krennic999** [comment]
@daiyixiang666 yes, the results are very unstable and there are some issues with generating some details

**Comment by krennic999** [comment]
@Ccioud err... currently we are addressing the issue of VAE, I think we can provide a usable solution by CVPR submission deadline.

**Comment by daiyixiang666** [comment]
What your opinion about the VAE, I think we can have some in depth discussion. The lower scale of the vae reconstruction is really bad

**Comment by daiyixiang666** [comment]
Do you think the share notebook in the VAE is important?

**Comment by krennic999** [comment]
@daiyixiang666, hi, you can send an email to xiao_xiao@mail.ustc.edu.cn and we can discuss further

**Comment by HalvesChen** [comment]
@krennic999 @Ccioud  Could you describe instability of var in detail? I've been doing experiments lately and I'm interested in it.

**Comment by Njasa2k** [comment]
@krennic999 any update on this project?

**Comment by krennic999** [comment]
😔I am very sorry, but our resources (data and gpus) are very limited, and we still haven't obtained good enough results. Maybe we will open source it in the future. If you are interested, you can pay attention to the two works of [HART](https://hanlab.mit.edu/projects/hart) and [infinity,](https://arxiv.org/html/2412.04431v1) which are similar research about VAR+t2i. Among them, infinity is the follow-up work of the original ByteDance VAR team, and HART uses a spatial interpolation of 2d RoPE similar to STAR.
